dtfs-central-api/api-tests/v1/utilisation-reports/get-utilisation-reports.api-test.ts

```diff
@@ -185,3 +185,173 @@ describe('GET /v1/bank/:bankId/utilisation-reports', () => {
     expect(response.body.length).toEqual(0);
   });
 });
+
+describe('GET /v1/utilisation-reports/:bankId/year', () => {
+  const getUrl = (bankId: string, year: string) => `/v1/utilisation-reports/${bankId}/${year}`;
+
+  const portalUser = {
+    _id: new ObjectId(),
+    firstname: 'Test',
+    surname: 'User',
+  } as PortalUser;
```

Comment:
I'm not a fan of `as`ing this, I know we do it in a fair few places but there is a `aPortalUser` test helper already defined which we can call to get a portal user and then just set the relevant fields on if they need to be set to specific values for the tests.